sdcm/nemesis.py

```diff
@@ -1688,6 +1691,76 @@ def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
                     finally:
                         clean_enospc_on_node(target_node=node, sleep_time=sleep_time)
 
+    def write_data_with_cs_to_reach_end_of_quota(self, node):
```

Comment:
I already commented long ago that this is not needed and what is need to be done is to write a sparse file as we do in ENOSPC nemesis.

Reply:
Not only it's not needed, it adds a load that some of the tests may not expect.

Reply:
Still relevant.